### DISCUSSION TITLE
XD5 Support?

### DISCUSSION THREAD

**Discussion by frostb1t** [EvanMulawski/FanControl.CorsairLink#180] [Ideas]
Hey,

Firstly I want to thank you for an awesome repo. I've been using this for a little bit to control my fan speed based on my XC7 waterblock temps. However, it'd be awesome if I could instead control them based on the coolant temperature that my Corsair XD5 pump reads instead.

From the looks of things, this isn't currently supported. I'm just wondering if this is something that is possible in the future?

Thanks again!

**Comment by EvanMulawski** [comment]
Moved to #181
